recipes/pdfmm/meta.yaml

```
@@ -0,0 +1,53 @@
+{% set name = "pdfmm" %}
+{% set version = "0.9.22" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/pdfmm/pdfmm/archive/refs/tags/v{{ version }}.tar.gz
+  sha256: 5883f4f1e01fb657f6b43a25115640292458c2b9db3f87d4517d2e341019a590
+
+build:
+  number: 0
+  skip: true  # [not linux]
```

```suggestion
build:
  number: 0
  skip: true  # [not linux]
  # TODO: Reassess pinning at version 1.0
  run_exports:
    - {{ pin_subpackage(name|lower, max_pin='x.x.x') }}
```
Run export with max_pin according to *ABI* compatability. Might be safe to pin to 'x.x' if there will be no ABI breaks except at MINOR versions.

---

thanks for the explanation! I have package for a long time here .. but `run_exports` is still something that I never understand well XD I will spend some time to understand it better! thank you so much!